google/cloud/language/CMakeLists.txt

```diff
@@ -112,7 +112,8 @@ target_include_directories(
 target_compile_options(google_cloud_cpp_language_mocks
                        INTERFACE ${GOOGLE_CLOUD_CPP_EXCEPTIONS_FLAG})
 
-if (BUILD_TESTING AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)
+include(CTest)
+if (BUILD_TESTING)
```

Comment:
The quickstart has a `catch(Status const&)`.  Now I wonder if the `noex` build is compiling the quickstarts.

Reply:
FWIW, I think we should leave the `AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS` and fix the build or all the CMake files in a separate PR.

Reply:
I messed up the re-basing, so I'm going to merge this and address your comment in a follow-up PR.